resources/assets/js/angular/app.js

```diff
@@ -86,11 +86,11 @@ angular.module('fc', [
       }) 
       .state('root.admin.privacy', {
          url: 'privacy',
-         templateUrl: 'angular/admin/privacy.tpl.html'
+         templateUrl: 'templates/admin/privacy.tpl.html'
       })
       .state('root.admin.contact', {
          url: 'contact',
-         templateUrl: 'angular/admin/contact.tpl.html'
+         templateUrl: 'templates/admin/contact.tpl.html'
       });
 })
 .run(['$rootScope', '$state', 'userService', function ($rootScope, $state, userService) {
```

Comment:
I don't particularly like this.  I want the js and the html (and I do think I want the css) to all be in the same spot for a particular module of the code.  Is there a reason it has to be this way?